### PR TITLE
Update WebUI quickstart

### PR DESCRIPTION
**Pull request by chtanch**
## Description

- Enlarge images and make them clickable to open them in new window
- Update model name in text to match image's model name
- Remove [image](https://llm-assets.readthedocs.io/en/latest/_images/webui_quickstart_load_model_error.png) for describing AttributeError since the image does not actually show that error. The AttributeError message is at the bottom and the entire traceback is too long to fit into one image.

### CODE REVIEW

**Comment by shane-huang** [comment]
also address this https://github.com/intel-analytics/BigDL/pull/10293#issuecomment-1972809034

**Comment by hkvision** [comment]
> also address this [#10293 (comment)](https://github.com/intel-analytics/BigDL/pull/10293#issuecomment-1972809034)

Can we just wait for the completion of adding warmup in code and directly ignore this for now?

**Comment by shane-huang** [comment]
> > also address this [#10293 (comment)](https://github.com/intel-analytics/BigDL/pull/10293#issuecomment-1972809034)
> 
> Can we just wait for the completion of adding warmup in code and directly ignore this for now?

Not sure if warm-up can cover all delay situations. Maybe still add a note in case?

**Comment by hkvision** [comment]
> > > also address this [#10293 (comment)](https://github.com/intel-analytics/BigDL/pull/10293#issuecomment-1972809034)
> > 
> > 
> > Can we just wait for the completion of adding warmup in code and directly ignore this for now?
> 
> Not sure if warm-up can cover all delay situations. Maybe still add a note in case?

Then let's add a quick remark as well? @chtanch

File: docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md [hunk 1]
```diff
@@ -104,6 +109,10 @@ To shut down the WebUI server, use **Ctrl+C** in the **Anaconda Prompt** termina
 
 ## Troubleshooting
 
+### Potentially slower first response
+
+The first response to user prompt might be slower than expected, with delays of up to several minutes before the response is generated. This delay occurs because the GPU kernels require compilation and initialization, which varies across different GPU models.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:114]: can we say ` delays of up to several minutes` here? @shane-huang

Comment (hkvision) [reply docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:114]: why different GPU models? should be different models or different GPUs?

File: docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md [hunk 2]
```diff
@@ -116,6 +125,8 @@ If there are still errors on missing packages, repeat the installation process f
...
 
 ### Compatiblity issues
-If you encounter **AttributeError** errors like shown below, it may be due to some models being incompatible with the current version of the transformers package because they are outdated. In such instances, using a more recent model is recommended.
-
-<img src="https://llm-assets.readthedocs.io/en/latest/_images/webui_quickstart_load_model_error.png" width=80%; />
+If you encounter **AttributeError** errors like `AttributeError: 'BaichuanTokenizer' object has no attribute 'sp_model'`, it may be due to some models being incompatible with the current version of the transformers package because the models are outdated. In such instances, using a more recent model is recommended.
+<!-- 
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/webui_quickstart_load_model_error.png">
+  <img src="https://llm-assets.readthedocs.io/en/latest/_images/webui_quickstart_load_model_error.png" width=100%; />
+</a> -->
```
Comment (hkvision) [inline docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:132]: These are commented?

File: docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md (hunk 1 above)
Comment (chtanch) [reply docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:114]: > can we say ` delays of up to several minutes` here? @shane-huang for windows quick start, also write like this: https://bigdl.readthedocs.io/en/latest/doc/LLM/Quickstart/install_windows_gpu.html#warmup-for-optimial-performance-on-first-run

Clarified with @Oscilloscope98, the delays are really that long for eg, iGPUs

> why different GPU models? should be different models or different GPUs?

I checked, 'GPU models' is the correct term, means 'GPU 型号'

Comment (jason-dai) [reply docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:114]: > I checked, 'GPU models' is the correct term, means 'GPU 型号'

Maybe change to "GPU types"?

Comment (hkvision) [reply docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md:114]: I also think GPU types is better. models is confusing in our project, mostly stand for llm models...

**Review by hkvision — APPROVED** [review]

**Pull request merged**